builder-k8s/action.go

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package builder
+package builder_k8s
```

Comment:
not sure if using underscore is a good practice, can it be renamed to avoid it?
instead builder-k8s, isn't better to use builder/k8s directory structure?

Reply:
Please see my comment above :P

Reply:
I've opened this PR from email and it sent me direct to the review page :)
didn't see that :)